### ISSUE TITLE
It's possible to create custom apps?

### ISSUE THREAD

**Issue by mgutbor** [revenz/Fenrus#67]
I need to include Joplin as app and I want to know how to do it. Thanks.

**Comment by revenz** [comment]
yes.  several community members have already done so.    There's lots of examples of "SmartApps" in that folder.  And the discord channel has a dedicated "devs" channel if you need help.
